timeseries/src/autogluon/timeseries/models/sktime/models.py

```diff
@@ -59,8 +194,79 @@ class AutoETSModel(AbstractSktimeModel):
         "random_state",
     ]
 
+    def _get_skt_forecaster_args(self, args: dict, min_length: int, inferred_period: int = 1):
+        fail_if_misoconfigured = args.pop("fail_if_misconfigured", False)
+
+        if min_length < 8 and args.get("initialization_method") == "heuristic":
+            error_message = f"{self.name}: Training series too short for initialization_method='heuristic'."
+            if fail_if_misoconfigured:
+                raise ValueError(error_message)
+            else:
+                warnings.warn(error_message + "\nFalling back to initialization_method='estimated'")
+                args["initialization_method"] = "estimated"
+
+        seasonality = args.pop("seasonality", "add")
+        if seasonality not in ["add", "mul", None]:
+            raise ValueError(
+                f"Invalid seasonality {seasonality} for model {self.name} (must be one of 'add', 'mul', None)"
+            )
+
+        seasonal_period = args.pop("seasonal_period", None)
+        # Handle the case when seasonal_period is intentionally set to None
```

Comment:
This is an explanation for what the code below does to remove the temptation to rewrite it as 
```python
        seasonal_period = args.pop("seasonal_period", inferred_period)
```
because this option won't handle the case when the user explicitly sets `seasonal_period=None` to perform inference automatically.

Reply:
I see. But the comment is a little confusing as the code below handles the case when `seasonal_period` is also _not_ intentionally (just, by default) set to None.